.github/scripts/deploy-appstarter

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+## Deploy codeigniter4/appstarter
+
+# Setup variables
+SOURCE=$1
+TARGET=$2
+RELEASE=$3
+
+echo "Preparing for version $3"
+echo "Merging files from $1 to $2"
+
+# Prepare the source
+cd $SOURCE
+git checkout master
+
+# Prepare the target
+cd $TARGET
+git checkout master
+rm -rf *
```

Comment:
This command scares me :) Can you explain it is doing. Seems like you checkout master branch then delete everything?

Reply:
The way the old script worked it would copy in the chosen files (like we do below) pre-wiping any directories (this was the bugged part from the `cprm` typo). Where that fails is when there are files that are renamed or removed (notice we have license.txt and a duplicate legacy LICENSE because of this). Using the modified version above it checks out the target repo (AppStarter or framework) and rebuilds the repo from scratch (* ignores files with leading periods so it doesn’t affect the `.git` directory). Also, it’s just a new commit so we could always back up one and nothing is lost!